torchaudio/backend/sox_io_backend.py

```diff
@@ -9,6 +9,9 @@
 import torchaudio
 from .common import AudioMetaData
 
+if _mod_utils.is_module_available('torchaudio._torchaudio'):
+    from torchaudio import _torchaudio  # noqa
```

Comment:
Could you try importing it after `_init_script_module(ext)` in the following location?

https://github.com/pytorch/audio/blob/23e9ed349f41fc2ec622855f554d4acc533b5bc2/torchaudio/extension/extension.py#L8-L13

like

```python
_init_script_module(ext) 
import torchaudio._torchaudio
```

It is nice if all the extension stuff is resolved in one palace.